### DISCUSSION TITLE
Why it's write "Sacré Coeur Vercel" ?

### DISCUSSION THREAD

**Discussion by Multigriffes** [elevedirect/app#5] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/90653170/207192817-685cd5ee-3422-4703-b51c-58e404de1ffc.png)

**Comment by camarm-dev** [comment, marked as answer]
This project is originally make for me and my friends... Our school use a special system (not the Ecoledirect one) for canteen's qr code cards so the showed card is only available for my établissement... I think I will add a field in config.ini to change canteen card provider (Ecoledirect or from student id). The problem is that my school does not use Ecoledirect integrate tool for cards so I have no ideas of what data Ecoledirect api send you back and where I can get these datas...
